### ISSUE TITLE
wthrr with WSL1 Windows 10

### ISSUE THREAD

**Issue by dtantono** [tobealive/wthrr-the-weathercrab#138]
Hi
I have a question: the wthrr installed properly on Ubuntu & Debian with Windows subsystem for Linux and I got nerd fonts as well. But, when I tried to run it. it throws this error messages (on Powershell it works though):  0: error trying to connect: dns error: failed to lookup address information: Temporary failure in name resolution
    1: dns error: failed to lookup address information: Temporary failure in name resolution
    2: failed to lookup address information: Temporary failure in name resolution
So, I suppose this wthrr does not work on WSL1. Is that so?
![image](https://github.com/tobealive/wthrr-the-weathercrab/assets/29157187/8a43c194-2502-4d08-a2a5-a6a7d9c733fd)

**Comment by kevinmatthes** [comment]
Is it possible to check the network settings for your WSL system?  Many hypervisors allow for the detailed configuration of the network settings of their maintained virtual machines.  And, as far as I know, WSL is just a virtual machine, as well, with Microsoft Windows as host and hypervisor.  If I remember correctly, settings regarding a WSL can be viewed and edited within a PowerShell using the appropriate command line instructions.

When you try to ask for your WSL's settings on a PowerShell, which output do you receive?

**Comment by dtantono** [comment]
Hi
Sorry it look like I use VPN and that issue comes up and when I turned off the VPN. it works again. I suppose, this app should work with VPN disabled. Is that so?

**Comment by tobealive** [comment]
Hey @dtantono,

sorry for the inconvenience. Also about weak support on windows, I left the windows eco system some while ago and rarely come back just for some bug fixes. 

Though VPN works for me on a linux based system there are definitely potential issues. But since you tried the auto location: the way wthrr currently works, it would try to provide the weather for the ip location of you VPN, which probably wouldn't be what your are looking for anyway? Could you try to specify a city in you call and check if this works for you?

Thanks alot @kevinmatthes for helping out!

**Comment by dtantono** [comment]
Hi
Thanks for your reply and I tried this wthrr -l en-gb 48.140232,11.558335 -f d,w or wthrr -l en-us London -f d,w those coordinates works without VPN. What do you think?

**Comment by dtantono** [comment]
not just London. but, everywhere with VPN cannot be used with WSL1 on Windows.

**Comment by ttytm** [comment]
Yes, it was meant to ask if requesting a city (not just london) via coordinates works with a VPN enabled, but requesting it by name does not. This would be something to investigate.
